redis/src/main/scala/zio/redis/api/Strings.scala

```diff
@@ -10,68 +10,235 @@ import zio.{ Chunk, ZIO }
-
+  /** Append a value to a key
+   *
+    * @param key Key of the string to add the value to
+   * @param value Value to append to the string
+   * @return Returns the length of the string after the append operation
+   */
+  final def append(key: String, value: String): ZIO[RedisExecutor, RedisError, Long] = Append.run((key, value))
+
+  /**
+   * Count set bits in a string
+   *
+    * @param key Key of the string of which to count the bits
+   * @param range Range of bytes to count
+   * @return Returns the number of bits set to 1
+   */
+  final def bitCount(key: String, range: Option[Range] = None): ZIO[RedisExecutor, RedisError, Long] =
+    BitCount.run((key, range))
+
+  /** Perform arbitrary bitfield integer operations on strings
+   *
+    * @param key Key of the string to operate on
+   * @param firstCommand First command to apply
+   * @param restCommands Subsequent commands to apply
+   * @return Returns an optional long result of each command applied
+   */
   final def bitField(
-    a: String,
-    b: BitFieldCommand,
-    bs: BitFieldCommand*
-  ): ZIO[RedisExecutor, RedisError, Chunk[Option[Long]]] = BitField.run((a, (b, bs.toList)))
-
-  final def bitOp(a: BitOperation, b: String, c: String, cs: String*): ZIO[RedisExecutor, RedisError, Long] =
-    BitOp.run((a, b, (c, cs.toList)))
-
-  final def bitPos(a: String, b: Boolean, c: Option[BitPosRange] = None): ZIO[RedisExecutor, RedisError, Long] =
-    BitPos.run((a, b, c))
-
-  final def decr(a: String): ZIO[RedisExecutor, RedisError, Long] = Decr.run(a)
-
-  final def decrBy(a: String, b: Long): ZIO[RedisExecutor, RedisError, Long] = DecrBy.run((a, b))
-
-  final def get(a: String): ZIO[RedisExecutor, RedisError, Option[String]] = Get.run(a)
-
-  final def getBit(a: String, b: Long): ZIO[RedisExecutor, RedisError, Long] = GetBit.run((a, b))
-
-  final def getRange(a: String, b: Range): ZIO[RedisExecutor, RedisError, String] = GetRange.run((a, b))
-
-  final def getSet(a: String, b: String): ZIO[RedisExecutor, RedisError, Option[String]] = GetSet.run((a, b))
-
-  final def incr(a: String): ZIO[RedisExecutor, RedisError, Long] = Incr.run(a)
-
-  final def incrBy(a: String, b: Long): ZIO[RedisExecutor, RedisError, Long] = IncrBy.run((a, b))
-
-  final def incrByFloat(a: String, b: Double): ZIO[RedisExecutor, RedisError, String] = IncrByFloat.run((a, b))
-
-  final def mGet(a: String, as: String*): ZIO[RedisExecutor, RedisError, Chunk[Option[String]]] =
-    MGet.run((a, as.toList))
-
-  final def mSet(a: (String, String), as: (String, String)*): ZIO[RedisExecutor, RedisError, Unit] =
-    MSet.run((a, as.toList))
-
-  final def mSetNx(a: (String, String), as: (String, String)*): ZIO[RedisExecutor, RedisError, Boolean] =
-    MSetNx.run((a, as.toList))
-
-  final def pSetEx(a: String, b: Duration, c: String): ZIO[RedisExecutor, RedisError, Unit] = PSetEx.run((a, b, c))
-
+    key: String,
+    firstCommand: BitFieldCommand,
+    restCommands: BitFieldCommand*
```

Comment:
I'd rename to `command` and `commands`.

Reply:
@anovakovic01 I like your naming suggestion more, but this style already passed some earlier [PR](https://github.com/zio/zio-redis/pull/185/files). @mijicd We should choose one style and stick to it?

Reply:
Good catch. I agree that we should stick to one style only. @mijicd should decide which one.

Reply:
I've followed the style of that merged PR, but I could change my PR.

Reply:
Let's go with the proposal from @anovakovic01

Reply:
I will update the names to remove the first and rest prefixes. I will need to open a new PR to do the same in the Sets API